odxtools/communicationparameter.py

```diff
@@ -33,7 +34,7 @@ def __str__(self) -> str:
         return f"CommunicationParameter('{self.id_ref}', value={val})"
 
     def _python_name(self):
-        return self.id_ref.replace(".", "__")
+        return self.id_ref.ref_id.replace(".", "__")
```

Comment:
I found some PDX files that use minus sign in the id string
```suggestion
        return self.id_ref.ref_id.replace(".", "__").replace("-", "_")
```

Reply:
hm, ok. I think that fixing this properly will require full internalization of the communication parameters (@smileprem is currently working on this, if I am not mistaken...) I'd say we should keep this in mind, but defer fixing this to another PR?

Reply:
Then revert `assert item_name.isidentifier()` in odxtools/nameditemlist.py until `_python_name` is covering all cases

Reply:
Hi @andlaus - Thanks for the trust on us. This is planned for Q4. I'll put up the MR in the next few weeks. But, in the meantime, please do what's needed to make the library usable. We could always replace the code. Just my opinion.

Reply:
> Then revert assert item_name.isidentifier() in odxtools/nameditemlist.py until _python_name is covering all cases

ah, I didn't make the connection here. That item names are identifiers is imperial for `NamedItemList` because the whole point of this class is to be able to access items via their name like e.g. `ecu.services.Default_Session_Start` instead of having to go through indices. I've amended the PR with your solution. (thanks for the catch.)